en/docs/install-and-setup/setup/api-controller/advanced-topics/configuring-environment-specific-parameters.md

```diff
@@ -75,6 +75,57 @@ When there are multiple artifacts which needs to be added as deployment-related
         </tbody>
     </table>
 
+
+## Bundling the generated directory before import
```

Comment:
Shall we capitalize each word in the title (except prepositions) because that convention is used in most of the pages for sub-topics? 
Like **"Bundling the Generated Directory before Import"**

Reply:
Fixed with b6dd23c33d3648b61e083aee1aee67878bebfb1d